docs/configuring-openshift.md

```diff
@@ -0,0 +1,79 @@
+# Configuring OpenShift
+The following sections describe how Minishift can be used to configure OpenShift. For details about managing the Minishift VM refer to the [Using Minishift](./using.md) section.
+
```

Comment:
IMO saying `how Minishift can be used to configure OpenShift` is wrong here. We can not configure any OpenShift using Minishift but the OpenShift instance started by Minishift. So the correct sentence would be `The following sections describe how the OpenShift instance started by Minishift can be configured`